app/src/pages/stakingOptions/stakingOptions.tsx

```diff
@@ -166,20 +169,28 @@ export const StakingOptions = (props: { network: string }) => {
 
   const soFilters: Array<any> = [
     {
-      text: 'CSA',
-      value: 'csa',
+      text: 'Bonus',
```

Comment:
Let's replace Bonus & Integration with "MM" in both tables. So you can easily view the rewards sent to all MMs (market makers)

Reply:
Selecting 'MM' now filters for any of 'mm', 'bonus', 'integration'. Users can still pick just Bonus or Integration.

![image](https://user-images.githubusercontent.com/96327301/222973325-8e80e7db-fc3b-4a8a-b014-5d00b5d79dcc.png)